uktrade/data-hub-frontend
# Update link contents/markup for better accessibility

## Description of change
As part of the accessibility fixes outlined in the DAC report this change will make a link within the list of companies more descriptive for screen readers. The content has changed and I have added `aria-label` with a detailed description to the link tag. 

## Test instructions
1. Run the mock stack
2. Go to add a new company from the companies collection list
3. Enter anything in the text field and search
4. The second item should have updated content and have an new aria attribute added to the link tag

## Screenshots
### Before
![Screenshot 2021-05-11 at 16 24 08](https://user-images.githubusercontent.com/10154302/117841949-61672700-b275-11eb-86b0-b9d956d50f58.png)

### After
![Screenshot 2021-05-11 at 16 13 55](https://user-images.githubusercontent.com/10154302/117841980-6af08f00-b275-11eb-8fa0-4a8b45dd5f59.png)
![Screenshot 2021-05-11 at 16 14 22](https://user-images.githubusercontent.com/10154302/117842004-704dd980-b275-11eb-9c96-0435100ab099.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
